tests/test_data.py

```diff
@@ -179,6 +179,45 @@ def test_looks_like_migrations_file_false(filename: str) -> None:
     assert not state.looks_like_migrations_file
 
 
+@pytest.mark.parametrize(
+    "filename",
+    (
+        "my_app/models/blog.py",
+        "my_app\\models\\blog.py",
```

Comment:
Let's test the two-deep case as well
```suggestion
        "my_app/models/blog.py",
        "my_app\\models\\blog.py",
        "my_app/models/blogging/blog.py",
        "my_app\\models\\blogging\\blog.py",
```